core-graphics/src/event.rs

```diff
@@ -43,9 +43,90 @@ bitflags! {
     }
 }
 
+/// These constants are the virtual keycodes defined originally in
+/// Inside Mac Volume V, pg. V-191. They identify physical keys on a
+/// keyboard. Those constants with "ANSI" in the name are labeled
+/// according to the key position on an ANSI-standard US keyboard.
+/// For example, `ANSI_A` indicates the virtual keycode for the key
+/// with the letter 'A' in the US keyboard layout. Other keyboard
+/// layouts may have the 'A' key label on a different physical key;
+/// in this case, pressing 'A' will generate a different virtual
+/// keycode.
+///
+/// [Ref](https://github.com/phracker/MacOSX-SDKs/blob/master/MacOSX10.13.sdk/System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/Headers/Events.h#L197-L261)
+#[repr(C)]
+pub struct ANSIKeyCode;
+impl KeyCode {
```

Comment:
Did you meant to write `impl AnsiKeyCode` here? In any case, I think you can just remove the new structures and keep only `KeyCode`, even though they are in separate enums in the Carbon headers, they are part of the same series of values. Separating seems unnecessary here.

Reply:
What do you want me to do with the doc comments? I think they add valuable information, but I don't know how to do that if they are not independent structs.  Should they get deleted or merged?

Reply:
The documentation could be combined into one block for `KeyCode` explaining where all these values come from in CoreGraphics.

Reply:
Okay, I combined the comments. Is this how you imagined it?